guides/source/caching_with_rails.md

```diff
@@ -512,12 +512,23 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
+### Notes on strong and weak ETags
+ETags generated by Rails are weak by default. Weak ETags allow semantically
+equivalent responses to have the same ETags, even if their bodies do not match
+exactly. This is useful when we don't want the page to be regenerated for minor
+changes in response body.
 
-Etags generated by Rails are weak by default. Weak etags allow semantically equivalent responses to have the same etags, even if their bodies do not match exactly. This is useful when we don't want the page to be regenerated for minor changes in response body. If you absolutely need to generate a strong etag, it can be assigned to the header directly.
 
 ```ruby
-  response.add_header "ETag", Digest::MD5.hexdigest(response.body)
+  response.weak_etag = response.body # => W/"hex string" → weak ETag
+```
```

Comment:
We don't need to show this as by default ETags generated by Rails are weak ETags.